packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -2738,27 +2738,36 @@ class Navigator extends StatefulWidget {
   /// the given context, and [Route.dispose] it.
   ///
   /// {@template flutter.widgets.navigator.removeRoute}
-  /// The removed route is removed without being completed, so this method does
-  /// not take a return value argument. No animations are run as a result of
-  /// this method call.
+  /// No animations are run as a result of this method call.
   ///
   /// The routes below and above the removed route are notified (see
   /// [Route.didChangeNext] and [Route.didChangePrevious]). If the [Navigator]
   /// has any [Navigator.observers], they will be notified as well (see
   /// [NavigatorObserver.didRemove]). The removed route is disposed without
   /// being notified. The future that had been returned from pushing that routes
-  /// will not complete.
+  /// will complete.
```

Comment:
I'm wondering what "removed route is disposed without being notified" means, specifically whether this sentence should still apply once we land this change.

Reply:
Route observers are aware of route removal so I didn't understand the purpose of this "notified" mention.

Reply:
After reading through this one more time, my guess would be that "without being notified" refers to not calling `didComplete`, since it's analogous to `didChangeNext` and `didChangePrevious` from the same paragraph.

With that in mind, we should probably remove that from the documentation to reflect this change.